README.md

```diff
@@ -13,5 +13,6 @@
 | 577 | Reverse Words in a String III | [View Solution](https://github.com/anurag629/LeetCode-Solution/tree/main/557.%20Reverse%20Words%20in%20a%20String%20III) |
 | 704 | Binary Search | [View Solution](https://github.com/anurag629/LeetCode-Solution/tree/main/704.%20Binary%20Search) |
 | 997 | Squares of a Sorted Array | [View Solution](https://github.com/anurag629/LeetCode-Solution/tree/main/977.%20Squares%20of%20a%20Sorted%20Array) |
-| ... | .. | .. |
```

Comment:
add this empty line(list) at the end

Reply:
Okay, should I make new PR?

Reply:
Oh, it's automatically updating into this PR xD